packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -78,263 +213,184 @@ const createEstate = async (estateContract, landContract, estate, wallet) => {
+
+  for (let asset of assetsToSell) {
+    const { nftAddress, id } = asset;
+    await placeAssetSellOrder(nftAddress, id);
   }
 };
 
-const placeParcelsSellOrders = async (
-  marketplace,
-  landAddress,
-  landIds,
-  priceInWei,
-  expireAt,
-  sellerWallet
-) => {
-  marketplace.setWallet(sellerWallet);
-  for (let assetId of landIds) {
-    const action = marketplace.createOrder(
-      landAddress,
-      assetId,
-      priceInWei,
-      expireAt,
-      gasParams
-    );
-    await action.waitForNonceToUpdate();
-  }
+const placeAssetSellOrder = async (nftAddress, assetId) => {
+  marketplaceContract.setWallet(sellerWallet);
+  const expireAt = Date.now() + duration.years(5);
+  const price = getRandomInt(10, 100) + "000";
+  const priceInWei = bigNumberify(price).mul(WeiPerEther);
+
+  const action = marketplaceContract.createOrder(
+    nftAddress,
+    assetId,
+    priceInWei,
+    expireAt,
+    gasParams
+  );
+  await action.waitForNonceToUpdate();
 };
 
-let network = process.env.NETWORK;
+const parcelsAreEqual = (p1, p2) => p1.x === p2.x && p1.y === p2.y;
 
-(async () => {
-  const config = require(`../config/${network}.js`);
-
-  // https://stats.goerli.net/
-  if (network === "goerli")
-    gasParams = {
-      gasLimit: 8e6,
-      gasPrice: 1e10,
-    };
-  else if (network === "development") {
-    network = "local";
-  }
-  const {
-    LANDProxy,
-    EstateRegistry,
-    Marketplace,
-    GnosisSafe: GnosisSafeInfo,
-    MANAToken,
-  } = TasitContracts[network];
-  const { address: MANA_ADDRESS } = MANAToken;
-  const { address: LAND_PROXY_ADDRESS } = LANDProxy;
-  const { address: ESTATE_ADDRESS } = EstateRegistry;
-  const { address: MARKETPLACE_ADDRESS } = Marketplace;
-  const { address: GNOSIS_SAFE_ADDRESS } = GnosisSafeInfo;
-
-  ConfigLoader.setConfig(config);
-
-  const [minterWallet, sellerWallet] = accounts;
-  const { address: sellerAddress } = sellerWallet;
-
-  const manaContract = new Mana(MANA_ADDRESS);
-  const landContract = new Land(LAND_PROXY_ADDRESS);
-  const estateContract = new Estate(ESTATE_ADDRESS);
-  const marketplaceContract = new Decentraland(MARKETPLACE_ADDRESS);
-  const gnosisSafeContract = new GnosisSafe(GNOSIS_SAFE_ADDRESS);
-
-  // Fund Gnosis Safe wallet with Mana tokens and ethers
-  const provider = ProviderFactory.getProvider();
-  await etherFaucet(provider, minterWallet, GNOSIS_SAFE_ADDRESS, ONE);
-  await erc20Faucet(
-    manaContract,
-    minterWallet,
-    GNOSIS_SAFE_ADDRESS,
-    ONE_THOUSAND
+const findParcel = (parcel, listOfParcels) => {
+  return listOfParcels.find(p => parcelsAreEqual(p, parcel));
+};
+
+const estateContainsParcelFromList = (estate, listOfParcels) => {
+  const { parcels: estateParcels } = estate;
+
+  for (let parcel of estateParcels)
+    if (findParcel(parcel, listOfParcels)) return true;
+
+  return false;
+};
+
+const getEstatesFromAPI = async () => {
+  const res = await fetch(
```

Comment:
Add a comment in the code about how we'll want to move away from this point of centralization in the future

Reply:
I should have made the comment one line lower so you could see that I mean the Decentraland API

Reply:
Done.

Reply:
Thanks! Resolving